docs/user/administration/users/member-types.md

```diff
@@ -0,0 +1,71 @@
+# BuddyPress Member Types
+
+BuddyPress Member Types feature allows for categorizing community members into distinct roles, enhancing the management and personalization of user experiences within the community.
+
+## Overview
+
+Member Types in BuddyPress provide a powerful way to classify users within your community. By defining various member types, such as "Students", "Teachers", or "Alumni", you can create a more organized and tailored community environment. This feature is especially useful for communities with diverse user groups, allowing customized interactions, content access, and privileges.
+
+## Creating Member Types
+
+Creating Member Types involves registering them within your BuddyPress theme or a custom plugin. This process allows you to define unique identifiers and labels for each member type, making them recognizable and manageable within the admin interface.
+
+### Add Member Type:
```

Comment:
```suggestion
### Add Member Type:

As explained earlier, you'll see into the screenshot below, the form to add your member type. Here are what inputs are used for within BuddyPress:

#### Member Type ID
 
It's your unique identifier for your member type. It's a required field and you need to use regular & lower-case character to fill this input. As this value is used as a fallback to the URL portion of the Member type directory link, make sure to choose wisely this identifier.
 
#### Singular name

This name will be shown into various places such as the user's metabox to set their member type (which is available into WP-Admin extended profile page) or the user's header of their front-end profile pages (if the **Show on Member** checkbox is activated).

#### Plural name

It's the plural form of the previous field. It's not currently used by BuddyPress, but it may be used by some plugins and themes.

#### Has Directory View

When this checkbox is active, you'll be able to filter the Members directory according to a specific member type.  FYI, a Member Type URL is looking like this `https://site.url/members/type/memberTypeID`. If you need to use a different URL portion to list users having as specific Member Type inside the Members directory, you can use the **Custom type directory slug** input that is appearing as soon as you activate the **Has Directory View** checkbox.

#### Show on Member

Activate this checkbox if you want the Member's Member Type information to be displayed on their front-end profile pages header.

```